landingzones/static/landingzones/js/landingzones.js

```diff
@@ -6,117 +6,118 @@ var isSuperuser = false;
  *****************************/
 var updateZoneStatus = function() {
     window.zoneStatusUpdated = false;
-    var zoneUuids = [];
+    var zoneUuids = '';
 
     $('.sodar-lz-zone-tr-existing').each(function() {
         var trId = $(this).attr('id');
         var zoneTr = $('#' + trId);
         var zoneUuid = $(this).attr('data-zone-uuid');
-        var sampleUrl = $(this).attr('data-sample-url');
         var statusTd = zoneTr.find('td#sodar-lz-zone-status-' + zoneUuid);
 
         if (statusTd.text() !== 'MOVED' && statusTd.text() !== 'DELETED') {
-            zoneUuids.push(zoneUuid);
+            zoneUuids += zoneUuid + ',';
```

Comment:
This latest update to the pr works now, and that's great. However, we should do this in a more standardized way.

The original way of sending this as a list is fine. The problems were that:
1. We were performing the request even when there was nothing to queryt for, and
2. The Ajax view didn't correctly handle not receiving a list.

Converting back and forth between strings adds extra complexity which we don't actually require.